x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -101,7 +102,36 @@ export const SearchExperience: React.FC = () => {
               view={SortingView}
             />
             <EuiSpacer />
-            <CustomizationCallout onClick={openCustomizationModal} />
+            {fields.filterFields.length > 0 ? (
+              <>
+                {fields.filterFields.map((fieldName) => (
+                  <div key={fieldName}>
```

Comment:
Minor semantic HTML suggestion

```suggestion
                  <section key={fieldName}>
```